src/runtime_src/core/include/experimental/xrt_ip.h

```diff
@@ -108,6 +109,16 @@ class ip : public detail::pimpl<ip_impl>
     XCL_DRIVER_DLLESPEC
     void
     wait();
+ 
+    /**
+     * wait() - Wait for interrupt or return on timeout
+     *
+     * Wait for interrupt from IP for timeout duration
+     * Timeout duration is in milliseconds
+     */
```

Comment:
Minor comment.  This is for doxygen and goes into online documentation, so commenting must be as accurate as possible.  

```suggestion
    /**
     * wait() - Wait for interrupt or timeout to occur
     *
     * @param timeout
     *   Timout in milliseconds.
     * @return
     *   std::cv_status::timeout if the timeout specified expired, 
     *   std::cv_status::no_timeout otherwise.
     * 
     * Blocks the current thread until an interrupt is received from the IP,  or 
     * until after the specified timeout duration
     */
```